.github/workflows/rust.yml

```diff
@@ -320,6 +320,20 @@ jobs:
       - name: Run datafusion-common tests
         run: cargo test -p datafusion-common --features=pyarrow
 
+  vendor:
+    name: Verify Vendored Code
+    runs-on: ubuntu-latest
+    container:
+      image: amd64/rust
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Rust toolchain
+        uses: ./.github/actions/setup-builder
+      - name: Run gen
+        run: ./regen.sh
+        working-directory: ./datafusion/proto
```

Comment:
This part shouldn't be necessary, as the script already handles this, but I suppose it can't hurt